src/views/studio/MyUploadsView/MyUploadsView.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react'
 
-import { useAuthorizedUser, useUploadsManager } from '@/hooks'
+import { useUploadsManager } from '@/hooks'
 
 import { AssetsGroupUploadBar } from './AssetsGroupUploadBar'
+import { AssetGroupUploadBarPlaceholder } from './AssetsGroupUploadBar/AssetGroupUploadBarPlaceholder'
 import { EmptyUploadsView } from './EmptyUploadsView'
 import { StyledText, UploadsContainer } from './MyUploadsView.style'
-import { placeholderItems } from './PlaceholderItems'
 
-const MyUploadsView = () => {
-  const { activeChannelId } = useAuthorizedUser()
-  const { uploadsState, isLoading } = useUploadsManager(activeChannelId)
+const MyUploadsView: React.FC = () => {
+  const { uploadsState, isLoading } = useUploadsManager()
 
   const hasUploads = uploadsState.length > 0
 
+  const placeholderItems = Array.from({ length: 5 }).map((_, idx) => <AssetGroupUploadBarPlaceholder key={idx} />)
```

Comment:
Those most likely are videos created on some different deployment? So for example on prod or other chain and the current chain doesn't know about those?

Reply:
I guess we could try to filter those out

Reply:
> Blank ones probably from another accounts

It seem like it's from another accounts. Same thing with the Channel group - there are 2 Channel groups on the screenshot. Normally all channel assets would stack into one Channel group since they would share the same `parentObjectId`. 

For the blank titles, even though assets would come from another channel, they still should be fetched properly?

Reply:
Yep, they should be fetched properly. Must like I said I think it may also be data from some other chain, since we may share local storage between those. Added code that will filter assets only owned by current channel